org.gnome.moserial.yaml

```diff
@@ -0,0 +1,30 @@
+id: org.gnome.moserial
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: moserial
+finish-args:
+  # moserial requires access to serial devices connected to the host.
+  - --device=all
+  - --filesystem=home
```

Comment:
what do you need full home directory access for that you couldn't get from the files portal? and/or a smaller scope.

Reply:
This was mostly a convenience for moserial user's since the application allows sending files over a serial console and receiving serial data to files in a directory, as well as saving settings files. Given there's no standard place for those types of files, I just used `--filesystem=home`. We might be able to get away with just `--filesystem=xdg-documents`, if we assume that's where most users save and load their files.

However, that wouldn't even be necessary using portals, right?
For a GTK application, does it just need to use the `NativeFileChooserDialog` and add `--talk-name=org.freedesktop.FileManager1`?

Reply:
GtkFileNativeChooser will use portals. You should also use XDG_CONFIG_HOME to store configuration and XDG_DATA_HOME can be used for other non-config data.